tests/integration/test_dynamodb.py

```diff
@@ -462,6 +462,42 @@ def test_empty_and_binary_values(self, dynamodb):
         response = table.put_item(Item=item2)
         assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
 
+    def test_batch_write_binary(self, dynamodb_client):
+        table_name = "table_batch_binary_%s" % short_uid()
+        dynamodb_client.create_table(
+            TableName=table_name,
+            AttributeDefinitions=[
+                {"AttributeName": "PK", "AttributeType": "S"},
+                {"AttributeName": "SK", "AttributeType": "S"},
+            ],
+            KeySchema=[
+                {"AttributeName": "PK", "KeyType": "HASH"},
+                {"AttributeName": "SK", "KeyType": "RANGE"},
+            ],
+            BillingMode="PAY_PER_REQUEST",
+        )
+        dynamodb_client.put_item(
+            TableName=table_name,
+            Item={"PK": {"S": "hello"}, "SK": {"S": "user"}, "data": {"B": b"test"}},
```

Comment:
Wondering if we should extend the tests to include non-decodable bytes here as well, e.g., `"data": {"B": b"test \xc0 \xed"}`